### PR TITLE
SHOT-4307/SHOT-4308: Modify start up logic to include python packages locally and remove Alias 2021 support

### PR DESCRIPTION
**Pull request by staceyoue**

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -65,3 +66,29 @@ make clean
...
+
+## Required Python Packages
+
+The framework requires additional Python packages to the standard package that ShotGrid Toolkit ships with. To make these packages available to the framework, they must be included in the distribution directory `dist/Python/Python<MAJOR_VERSION><MINOR_VERSION>/packages`.
+
+### What's included?
+
+The packages that are included are defined by the `requirements.txt` in each of the `dist/Python/Python<MAJOR_VERSION><MINOR_VERSION>` directory. The packages can be obtained by running the [update_python_packages.py](https://github.com/shotgunsoftware/tk-framework-alias/blob/ticket/SHOT-4307/dev/update_python_packages.py) script that uses pip to intall them. The script will then zip of the packages and copy them to the distribution directory. Standard Python packages will be found in `pkgs.zip`, and C extension packages will be found in `c_extensions.zip`. The reason for separating these packages is because C extension packages need to be unzipped in order to allow them to be imported by the framework (while standard packages do not).
```
Comment (rob-aitchison) [inline dev/README.md:76]: Will these github links break once merged and the branch is deleted?

File: dist/README.md [hunk 2]
```diff
@@ -59,8 +64,23 @@ websocket-client
...
 
 If the embeddable package needs to be updated, for example, to add more site-packages, remember to update the `embed_version.txt` file to bump the version. This version is checked to see if the installed embedded package needs to be updated on the user's local disk.
+
+## Required Python Packages
+
+Additional Python packages are required to run the framework that ShotGrid Toolkit does not ship with. These packages are added to the Python system path at run time [here](https://github.com/shotgunsoftware/tk-framework-alias/blob/aea3b00153d1f0e63c38b4444fa24764fbc6dcfa/python/tk_framework_alias/__init__.py#L11-L20). By including the packages in the path, these modules must be made available by either:
```
Comment (rob-aitchison) [inline dist/README.md:75]: This one looks like a permalink bound to a commit IIRC.

**Review by rob-aitchison — COMMENTED** [review]
Looking great functionally, thanks!
Just a question about permalinks in the URL's I flagged ( a couple of them only.)

File: dev/README.md (hunk 1 above)
Comment (staceyoue) [reply dev/README.md]: Good catch. I updated all links to point to the default (develop) branch

**Review by rob-aitchison — APPROVED** [review]
Looking great, thanks!

**Pull request merged**